_includes/getting-started.html

```diff
@@ -63,44 +98,84 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
             <!-- Kubernetes - Helm -->
             <h4 style="font-size: 1.5em;">Using Helm</h4>
             <p>Helm v3 - run the following:</p>
-            <div class="highlight-code highlight-source-shell">
-              <pre><code>
-  $ git clone https://github.com/layer5io/meshery.git; cd meshery
-  $ kubectl create namespace meshery
-  $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
-              </code></pre>
+            <div id="start-brew" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
```

Comment:
The div id must be unique. Should be something like `start-helm-v3`. This id needs to match the `data-clipboard-target` value on line 110.